system/Commands/Sessions/Views/migration.tpl.php

```diff
@@ -38,30 +38,50 @@
 
 use CodeIgniter\Database\Migration;
 
-class Migration_create_sessions_table extends Migration
+class Migration_create_<?= $tableName ?>_table extends Migration
 {
+<?php if (isset($DBGroup)) : ?>
+	protected $DBGroup = '<?= $DBGroup ?>';
+<?php endif ?>
+
 	public function up()
 	{
 		$this->forge->addField([
-				'id'         => ['type' => 'INT', 'constraint' => 9, 'unsigned' => true, 'auto_increment' => true],
-				'ip_address' => ['type' => 'VARCHAR', 'constraint' => 45, 'null' => false],
-				'timestamp'  => ['type' => 'INT', 'constraint' => 10, 'unsigned' => true, 'null' => false, 'default' => 0],
-				'data'       => ['type' => 'text', 'null' => false, 'default' => ''],
+			'id'         => [
+				'type'       => 'VARCHAR',
+				'constraint' => 128,
+				'null'       => FALSE
```

Comment:
Per the styleguide, all boolean values (and other types) should be lowercase (like true, false, null, etc)